docs/cloud-native-security/cspm-get-started-azure.asciidoc

```diff
@@ -0,0 +1,121 @@
+[[cspm-get-started-azure]]
+= Get started with CSPM for Azure
+
+[discrete]
+[[cspm-overview-azure]]
+== Overview
+
+This page explains how to get started monitoring the security posture of your cloud assets using the Cloud Security Posture Management (CSPM) feature.
+
+.Requirements
+[sidebar]
+--
+* The CSPM integration is available to all {ecloud} users. On-premise deployments require an https://www.elastic.co/pricing[Enterprise subscription].
+* To view posture data, you need `read` privileges for the following {es} indices:
+** `logs-cloud_security_posture.findings_latest-*`
+** `logs-cloud_security_posture.scores-*`
+** `logs-cloud_security_posture.findings`
+* The user who gives the CSPM integration permissions must be an Azure subscription `admin`.
+--
+
+[discrete]
+[[cspm-setup-azure]]
+== Set up CSPM for Azure
+
+To set up CSPM for Azure, first add the CSPM integration, then enable cloud account access.
+
+
+[discrete]
+[[cspm-add-and-name-integration-azure]]
+=== Add your CSPM integration
+. From the Elastic Security *Get started* page, click *Add integrations*.
+. Search for `CSPM`, then click on the result.
+. Click *Add Cloud Security Posture Management (CSPM)*.
+. Under **Configure integration**, select **Azure**.
+. Give your integration a name that matches the purpose or team of the Azure subscription you want to monitor, for example, `azure-dev-policy`.
+
+[discrete]
+[[cspm-set-up-cloud-access-section-azure]]
+=== Set up cloud account access
+
+NOTE: To set up CSPM for an Azure subscription, you will need admin privileges for that subscription.
+
+For most users, the simplest option is to use an Azure Resource Manager (ARM) template to automatically provision the necessary resources and permissions in Azure. If you prefer a more hands-on approach or require a specific configuration not supported by the ARM template, you can use one of manual setup options described below.
+
+[discrete]
+[[cspm-set-up-ARM]]
+== ARM template setup (recommended)
+
+. Under *Setup Access*, select *ARM Template*.
+. Under **Where to add this integration**:
+.. Select **New Hosts**.
+.. Name the {agent} policy. Use a name that matches the purpose or team of the cloud account or accounts you want to monitor. For example, `azure-dev-policy`. Click **Save and continue**. The *ARM Template deployment* window appears.
+.. In a new tab, log in to the Azure portal, then return to {kib} and click **Launch ARM Template**. This will open the ARM template in Azure.
+.. (Optional) Change the `Resource Group Name` parameter. Otherwise the name of the resource group defaults to `cloudbeat-` and a timestamp.
```

Comment:
```suggestion
.. (Optional) Change the `Resource Group Name` parameter. Otherwise the name of the resource group defaults to a timestamp prefixed with `cloudbeat-`.
```